test/Concurrency/closure_isolation.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -dump-ast %s  -disable-availability-checking | %FileCheck %s
+// RUN: %target-swift-frontend -dump-ast %s  -disable-availability-checking -enable-experimental-feature ClosureIsolation | %FileCheck %s
```

Comment:
Since you added `-enable-experimental-feature` this test-case now needs `// REQUIRES: asserts`